### ISSUE TITLE
lower accuracy when finetuning on Physio dataset

### ISSUE THREAD

**Issue by sigureling** [wjq-learning/CBraMod#17]
Hi. I preprocess Physionet dataset using preprocessing_physio.py. And I pass the following hyper-parameter which is from Cbramod paper Appendix D.4. 
Namespace(seed=60, cuda=2, epochs=50, batch_size=64, lr=0.0001, weight_decay=0.05, optimizer='AdamW', clip_value=1, dropout=0.1, classifier='all_patch_reps', downstream_dataset='PhysioNet-MI', datasets_dir=xxx', num_of_classes=4, model_dir=xxx', num_workers=16, label_smoothing=0.1, multi_lr=True, frozen=False, use_pretrained_weights=True, foundation_dir='cbramod.pt')

I change the seed for 5 times. But I still find the result to be lower. Can u offer me more information to get the ideal result? Thank you.
<img width="1341" height="960" alt="Image" src="https://github.com/user-attachments/assets/acf1d21b-78b7-4076-b1d9-37f653f20334" />
